aws/logs_monitoring/lambda_function.py

```diff
@@ -122,11 +122,34 @@ def __init__(self, name, pattern, placeholder):
             "(?<!^)\s+(?={})(?!.\s)".format(DD_MULTILINE_LOG_REGEX_PATTERN)
         )
     except Exception:
-        raise Exception("could not compile multiline regex with pattern: {}".format(DD_MULTILINE_LOG_REGEX_PATTERN))
+        raise Exception("could not compile s3 multiline regex with pattern: {}".format(DD_MULTILINE_LOG_REGEX_PATTERN))
     multiline_regex_start_pattern = re.compile(
         "^{}".format(DD_MULTILINE_LOG_REGEX_PATTERN)
     )
 
+
+# DD_MULTILINE_CLOUDWATCH_REGEX: Datadog Multiline Cloudwatch Logs Regular Expression Patterns for Log Groups
+DD_MULTILINE_CLOUDWATCH_LOG_REGEX_PATTERNS = None
+if "DD_MULTILINE_CLOUDWATCH_LOG_REGEX_PATTERNS" in os.environ:
+    multiline_regexs = os.environ["DD_MULTILINE_CLOUDWATCH_LOG_REGEX_PATTERNS"]
+    try:
+        DD_MULTILINE_CLOUDWATCH_LOG_REGEX_PATTERNS = {}
+        cloudwatch_regex_dict = json.loads(multiline_regexs)
+
+        # compile regex start paterns for each dictionary entry
+        for log_group_name in cloudwatch_regex_dict:
```

Comment:
do we want to use `items()` do have both the `group_name` and the `rule` ?

Reply:
👍 ty, updated to use `iteritems` (for 2.7)